config/database.yml

```diff
@@ -0,0 +1,17 @@
+# default: &default
```

Comment:
don't check-in comments. They are already in your source history in git so you don't need to. Keep everything clean and focused.